src/server/scripts/Kalimdor/CavernsOfTime/BattleForMountHyjal/boss_archimonde.cpp

```diff
@@ -4,19 +4,20 @@
  * Copyright (C) 2005-2009 MaNGOS <http://getmangos.com/>
  */
 
-/* ScriptData
-SDName: Boss_Archimonde
-SD%Complete: 85
-SDComment: Doomfires not completely offlike due to core limitations for random moving. Tyrande and second phase not fully implemented.
-SDCategory: Caverns of Time, Mount Hyjal
-EndScriptData */
+ /* ScriptData
+ SDName: Boss_Archimonde
+ SD%Complete: 85
+ SDComment: Doomfires not completely offlike due to core limitations for random moving. Tyrande and second phase not fully implemented.
+ SDCategory: Caverns of Time, Mount Hyjal
+ EndScriptData */
 
 #include "ScriptMgr.h"
 #include "ScriptedCreature.h"
 #include "hyjal.h"
```

Comment:
You mean combine them?

Reply:
Yes, I think they could be combined. But if you want to keep that out of scope, just resolve this item. Totally up to you :smile:

Reply:
An issue for another time! 🙂

Reply:
I know is not part of your PR, but could you rename or reorganize these files please?
`hyjal.h`
`hyjal_trash.h`